OverlayPlugin.Core/NetworkProcessors/LineActorControlCommon.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace RainbowMage.OverlayPlugin.NetworkProcessors
+{
+    public enum Server_ActorControlCategory : ushort
+    {
+        SetAnimationState = 0x003E, // 62
+        StatusUpdate = 0x01F8, // 504
+        // Name is a guess
+        DisplayLogMessage = 0x020F, // 527
+        DisplayLogMessageParams = 0x0210, // 528
```

Comment:
What's included in this category?  I see it in `AllowedActorControlCategories` below, but I don't recall whether this was something we talked about (and I'm just forgetting, entirely likely) or if this is something new you found?

Reply:
I added an additional column to `LineActorControlSelfExtra` to explain why this is included:
```cs
            // Some `LogMessage` messages can be triggered by both 0x020F and 0x0210 categories, not sure what the difference is
            // except that 0x0210 messages usually have another actor ID in the parameters
```

Hopefully that makes sense?